backend.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  backend "s3" {
+    bucket         = "rss-tf-state"
+    key            = "rss/terraform.tfstate"
+    region         = "eu-west-1"
+    encrypt        = true
+    dynamodb_table = "rss-tf-state-lock"
```

Comment:
Using DynamoDB table for state locking is deprecaded now.
[link](https://medium.com/aws-specialists/dynamodb-not-needed-for-terraform-state-locking-in-s3-anymore-29a8054fc0e9)
It's not a mistake, just FYI.